src/output/RepresenterRewriteOutput.ts

```diff
@@ -29,7 +31,18 @@ export class RepresenterRewriteOutput implements Output {
     getProcessLogger().log(JSON.stringify(this.representation, undefined, 2))
     const normalized = normalizeRepresentation(this.representation)
 
-    const representation = generate(normalized.representation, {})
+
+    const quotesFormatGenerator = Object.assign({}, GENERATOR, {
+      Literal: function (node: LiteralNode, state: State) {
+        const { type, value } = node
+        const quote = '`';
+        if (type?.startsWith('Lit') && typeof value === 'string') {
```

Comment:
About the type test, I could do `type === 'Literal'`. There isn't a 'StringLiteral' or anything of the sort, that's why i'm also testing for `typeof value === 'string'`

About the tests, I realize that some of the passing tests will probably fail now, because `\fixtures\two-fer\pass\expected-representation.txt` is no longer correct. I can change it, however, Getting the test suite to run in Windows is a pain.

Also, Is the test for `generates a representation.txt` of the passing tests correct, or I don't get something? Asking because i'm getting some errors for invalid JSON parse.

PS: Really sorry that it's taking this amount of effort....

Reply:
> About the type check, I could do type === 'Literal'.

Yes I'd prefer that.

Reply:
> If you approve, I'll edit the expected-representations to use backticks instead of single quotes.

Please do!

Reply:
> About the type check, I could do type === 'Literal'. 

I don't see this change?